mycli/key_bindings.py

```diff
@@ -76,10 +76,24 @@ def _(event):
         b = event.app.current_buffer
         b.complete_state = None
 
+    @kb.add('c-j')
+    def _(event):
+        """Introduces a line break in Emacs mode, regardless of multi-line
+        mode or not."""
+        _logger.debug('Detected control-j key.')
+        if event.app.editing_mode == EditingMode.EMACS:
+            event.app.current_buffer.insert_text('\n')
```

Comment:
Why `\n` and not `os.linesep`?

Reply:
I don't understand the platform-specific subtleties well enough to be confident which way is correct.  Here I just copied the existing `'\n'`.